diff --git a/cmake/defaults/Options.cmake b/cmake/defaults/Options.cmake
index bcda89885..3cb541885 100644
--- a/cmake/defaults/Options.cmake
+++ b/cmake/defaults/Options.cmake
@@ -25,6 +25,7 @@ option(PXR_STRICT_BUILD_MODE "Turn on additional warnings. Enforce all warnings
 option(PXR_VALIDATE_GENERATED_CODE "Validate script generated code" OFF)
 option(PXR_HEADLESS_TEST_MODE "Disallow GUI based tests, useful for running under headless CI systems." OFF)
 option(PXR_BUILD_TESTS "Build tests" ON)
+option(PXR_BUILD_USD_TOOLS "Build commandline tools" ON)
 option(PXR_BUILD_IMAGING "Build imaging components" ON)
 option(PXR_BUILD_EMBREE_PLUGIN "Build embree imaging plugin" OFF)
 option(PXR_BUILD_OPENIMAGEIO_PLUGIN "Build OpenImageIO plugin" OFF)
diff --git a/cmake/defaults/Packages.cmake b/cmake/defaults/Packages.cmake
index 9be215b36..0ce6e6fd5 100644
--- a/cmake/defaults/Packages.cmake
+++ b/cmake/defaults/Packages.cmake
@@ -64,7 +64,7 @@ else()
 endif()
 
 # --TBB
-find_package(TBB REQUIRED COMPONENTS tbb)
+find_package(TBB)
 add_definitions(${TBB_DEFINITIONS})
 
 # --math
diff --git a/pxr/base/lib/plug/initConfig.cpp b/pxr/base/lib/plug/initConfig.cpp
index 91d01cadc..61bb6de84 100644
--- a/pxr/base/lib/plug/initConfig.cpp
+++ b/pxr/base/lib/plug/initConfig.cpp
@@ -33,6 +33,7 @@
 #include "pxr/base/arch/symbols.h"
 
 #include <boost/preprocessor/stringize.hpp>
+#include <boost/dll/runtime_symbol_info.hpp>
 
 PXR_NAMESPACE_OPEN_SCOPE
 
@@ -83,6 +84,20 @@ ARCH_CONSTRUCTOR(Plug_InitConfig, 2, void)
 
     sharedLibPath = TfGetPathName(sharedLibPath);
 
+    // Executable location, which is relevant when using static linking.
+    boost::filesystem::path executablePath = boost::dll::program_location();
+    if (!executablePath.empty()) {
+        boost::filesystem::path pluginPath;
+
+        // The path MUST end in a slash, or symlinks will not be treated as directory.
+        // Two paths are added, one for relative to Blender, one for relative to unit tests.
+        pluginPath = executablePath.parent_path() / "lib/usd/";
+        result.push_back(pluginPath.string());
+
+        pluginPath = executablePath.parent_path().parent_path() / "lib/usd/";
+        result.push_back(pluginPath.string());
+    }
+
     // Environment locations.
     _AppendPathList(&result, TfGetenv(pathEnvVarName), sharedLibPath);
 
@@ -94,6 +109,13 @@ ARCH_CONSTRUCTOR(Plug_InitConfig, 2, void)
     _AppendPathList(&result, installLocation, sharedLibPath);
 #endif // PXR_INSTALL_LOCATION
 
+    if (!TfGetenv("PXR_PATH_DEBUG").empty()) {
+        printf("USD Plugin paths: (%zu in total):\n", result.size());
+        for(const std::string &path : result) {
+            printf("    %s\n", path.c_str());
+        }
+    }
+
     Plug_SetPaths(result);
 }
 
diff --git a/pxr/usd/CMakeLists.txt b/pxr/usd/CMakeLists.txt
index c894cca37..df087a418 100644
--- a/pxr/usd/CMakeLists.txt
+++ b/pxr/usd/CMakeLists.txt
@@ -1,6 +1,5 @@
 set(DIRS
     lib
-    bin
     plugin
 )
 
@@ -8,3 +7,8 @@ foreach(d ${DIRS})
     add_subdirectory(${d})
 endforeach()
 
+if (PXR_BUILD_USD_TOOLS)
+    add_subdirectory(bin)
+else()
+    message(STATUS "Skipping commandline tools because PXR_BUILD_USD_TOOLS=OFF")
+endif()
